src/main/java/org/folio/rest/impl/PatronActionSessionAPI.java

```diff
@@ -55,6 +87,45 @@ public void deletePatronActionSessionStoragePatronActionSessionsByPatronSessionI
       DeletePatronActionSessionStoragePatronActionSessionsByPatronSessionIdResponse.class, asyncResultHandler);
   }
 
+  @Override
+  public void getPatronActionSessionStorageExpiredSessionPatronIds(
+    PatronActionSessionStorageExpiredSessionPatronIdsGetActionType actionType, String lastTimeActionLimit,
+    int limit, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    String tenantId = okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT);
+    PgClientFutureAdapter pgClient = PgClientFutureAdapter.create(vertxContext, okapiHeaders);
+    DateTime dateTimeLimit;
+    try {
+      dateTimeLimit = DateTime.parse(lastTimeActionLimit);
+    }
```

Comment:
Could this be at the  same line?
"} catch () {"

Reply:
updated